x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/header_actions/search_engines_playground.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiButton } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { ENTERPRISE_SEARCH_CONTENT_PLUGIN } from '../../../../../../../common/constants';
+import { KibanaLogic } from '../../../../../shared/kibana';
+
+import { PLAYGROUND_PATH } from '../../../../routes';
+
+export interface SearchEnginesPlaygroundProps {
+  indexName: string;
+  ingestionMethod: string;
+}
+
+export const SearchEnginesPlayground: React.FC<SearchEnginesPlaygroundProps> = ({
+  indexName,
+  ingestionMethod,
+}) => {
+  const playgroundUrl =
+    ENTERPRISE_SEARCH_CONTENT_PLUGIN.URL + PLAYGROUND_PATH + `?defaultIndexName=${indexName}`;
```

Comment:
PLAYGROUND_PATH should already contain the ENTERPRISE_SEARCH_CONTENT_PLUGIN url, because it always lives there. If it doesn't we should correct that--you should be able to just reference the playground path in isolation to navigate to it. See how URLs are defined here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/routes.ts

Also, have you checked that this works when Kibana runs with a prefix (it should run that way by default)?

Reply:
Got it, `Search Application` defined differently as it takes us outside of `content`. Thank you :)